packages/browser-integration-tests/src/segment-retries.test.ts

```diff
@@ -1,30 +1,38 @@
 import { Request, test, expect } from '@playwright/test'
 import { SettingsBuilder } from './fixtures/settings'
 import { standaloneMock } from './helpers/standalone-mock'
+import { extractWriteKeyFromUrl } from './helpers/extract-writekey'
+import {
+  PersistedQueueResult,
+  getPersistedItems,
+} from './helpers/get-persisted-items'
 
 test.describe('Standalone tests', () => {
```

Comment:
Was this meant to be double nested in here?

Reply:
Ooph, I think it was meant to signify these tests are for the standalone (CDN) version of a.js, not the NPM version. Can probably remove this describe block though.